src/main/java/uk/gov/hmcts/darts/Application.java

```diff
@@ -26,4 +28,9 @@ public static void main(final String[] args) {
         SpringApplication.run(Application.class, args);
     }
 
+    @Bean
```

Comment:
I think the convention currently is to add a `Configuration` class under a module e.g. `audit.config` to hold our custom beans. Alternatively we could just specify `@Component` on the `AuditSearchQueryValidator` class

Reply:
ACCEPT